tests/mocks/sublime_mock.py

```diff
@@ -29,3 +29,11 @@ def load_settings(*args):
         'included_files': [],
         'excluded_files': []
     })
+
+
+def encode_value(*args):
+    pass
+
+
+def decode_value(*args):
```

Comment:
replace this function by:

```python
def decode_value(content):
    decoded = re.sub(re.compile("/\*.*?\*/",re.DOTALL ) ,"" ,content)
    return json.loads(re.sub(re.compile("//.*?\n" ) ,"" ,decoded))
```


With this you can replicate the behavior of "sublime.decode_value"

Reply:
this function removes comments from any json content.

Reply:
Can you add a new test validating this behavior?

something like this.

```python
    @mock.patch('sync_settings.libs.path.exists', mock.MagicMock(return_value=True))
    @mock.patch('sync_settings.sync_version.open', mock.mock_open(
        read_data='{"created_at": "2019-01-11T02:15:15Z", /* some comment */"hash": "123123123"}'))
    def test_get_local_version_with_commented_content(self):
        v = version.get_local_version()
        self.assertDictEqual({'hash': '123123123', 'created_at': '2019-01-11T02:15:15Z'}, v)
```